features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/ParserBase.java

```diff
@@ -328,7 +333,7 @@ protected CompletableFuture<?> transmit(final RecordProvider packet, final Sessi
                         // Let's serialize
                         final FlowMessage.Builder flowMessage;
                         try {
-                            flowMessage = this.getMessageBuilder().buildMessage(record, enrichment);
+                            flowMessage = transformRawMessage(includeRawMessage, this.getMessageBuilder().buildMessage(record, enrichment), record);
```

Comment:
You can just inline it, work on the builder directly and maybe even move this out of the try-catch-block?

Reply:
done